contracts/modules/factories/ModuleFactory.sol

```diff
@@ -6,12 +6,20 @@ import {Spigot} from "../spigot/Spigot.sol";
 import {Escrow} from "../escrow/Escrow.sol";
 
 contract Factory is IModuleFactory {    
+
+    Spigot spigot;
+    Escrow escrow;
+
     function DeploySpigot(address owner, address treasury, address operator) external returns (address){
```

Comment:
idk when these function names became CapitalCase but can u convert them to camelCase in the contract and interface